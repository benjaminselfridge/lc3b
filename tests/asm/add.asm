        0x3000
        ADD R0  R0  0x9
        ADD R1  R0  R0
        ADD R2  R1  R0
        TRAP 0x0
