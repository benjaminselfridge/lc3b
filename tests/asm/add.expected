[ RegContains 0 9
, RegContains 1 18
, RegContains 2 27
]